src/dsql/parse-conflicts.txt

```diff
@@ -1 +1 @@
-117 shift/reduce conflicts, 22 reduce/reduce conflicts.
+130 shift/reduce conflicts, 13 reduce/reduce conflicts.
```

Comment:
Quite a lot of new shift/reduce conflicts, did you analyse whether the new syntax ambiguities safe?

Reply:
And quite a lot decrease in the more problematic reduce/reduce conflicts ;)
Do you really think it's important to track this? It was not an explosion of conflicts. This is difficult to track and report details. What I can "easily" say is that this is split in some conflicts per some of new rules.